src/biokbase/narrative/jobs/appmanager.py

```diff
@@ -4,7 +4,7 @@
 import biokbase.auth as auth
 from .job import Job
 from .jobmanager import JobManager
-from .jobcomm import JobComm, exc_to_msg, RUN_STATUS, NEW
+from .jobcomm import JobComm, exc_to_msg, MESSAGE_TYPE
```

Comment:
Coming up: fairly large number of search/replaces substitutions from referring to messages as `MESSAGE_TYPE["msg_name"]` rather than `MSG_NAME`.